app/models/miq_report.rb

```diff
@@ -223,4 +225,20 @@ def remove_loading_relations_for_virtual_custom_attributes
     vc_attributes = CustomAttributeMixin.select_virtual_custom_attributes(cols).present?
     include.delete(:custom_attributes) if vc_attributes.present? && include && include[:custom_attributes].blank?
   end
+
```

Comment:
comment added, but I moved excluding columns to the after_initialize,
because before array operation `.push` does not work as I expected.

```
a = MiqReport.last
a.cols.push("aaa")
```